subtrack.MAUI/Services/SubscriptionService.cs

```diff
@@ -2,6 +2,8 @@
 using subtrack.DAL;
 using subtrack.DAL.Entities;
 using subtrack.MAUI.Services.Abstractions;
+using System.Linq;
+using static Android.Content.Res.Resources;
```

Comment:
Shouldnt use android related things in our services. If its for the exception, create a custom exception https://learn.microsoft.com/en-us/dotnet/standard/exceptions/how-to-create-localized-exception-messages